### ISSUE TITLE
Loss increasing after 1000 steps in training

### ISSUE THREAD

**Issue by ekesdf** [armaanpriyadarshan/Training-a-Custom-TensorFlow-2.X-Object-Detector#18]
INFO:tensorflow:Step 500 per-step time 3.462s loss=0.294
I0105 18:34:59.967795 140369640556352 model_lib_v2.py:651] Step 500 per-step time 3.462s loss=0.294
INFO:tensorflow:Step 600 per-step time 3.506s loss=0.335
I0105 18:40:52.187162 140369640556352 model_lib_v2.py:651] Step 600 per-step time 3.506s loss=0.335
INFO:tensorflow:Step 700 per-step time 3.516s loss=0.376
I0105 18:46:43.798759 140369640556352 model_lib_v2.py:651] Step 700 per-step time 3.516s loss=0.376
INFO:tensorflow:Step 800 per-step time 3.497s loss=0.564
I0105 18:52:36.506826 140369640556352 model_lib_v2.py:651] Step 800 per-step time 3.497s loss=0.564
INFO:tensorflow:Step 900 per-step time 3.589s loss=1.881
I0105 18:58:28.063675 140369640556352 model_lib_v2.py:651] Step 900 per-step time 3.589s loss=1.881
INFO:tensorflow:Step 1000 per-step time 3.605s loss=4.473
I0105 19:04:19.890992 140369640556352 model_lib_v2.py:651] Step 1000 per-step time 3.605s loss=4.473
INFO:tensorflow:Step 1100 per-step time 3.562s loss=9.153
I0105 19:10:13.496691 140369640556352 model_lib_v2.py:651] Step 1100 per-step time 3.562s loss=9.153

**Comment by armaanpriyadarshan** [comment]
This is weird, what is your dataset looking like?

**Comment by ekesdf** [comment]
it is a dataset with 832x832 images
where each image is a grid of 8x8 of handwritten chars in the size between 20x20 by 80x80

**Comment by armaanpriyadarshan** [comment]
From you're description, it seems you might want to look into OCR instead of Object Detection. OCR is Optical Character Recognition. I have limited experience, but I would recommend trying out Tesseract OCR.

**Comment by ekesdf** [comment]
I have tried all different types of OCR but none of them worked well for my use case, therefore, I am trying to do it my own way
with simple Object detection. But I'm having problems with small objects like if I want to upscale the input size to say like a 1024x1024 with the same size of char but the detection result are far from that what I achieved on the small scale network

**Comment by armaanpriyadarshan** [comment]
In the pipeline.config, you might want to adjust a few things. In accordance with your image resolution, change the image_resizer from 640 x 640 to your specific resolution. You might also want to increase the epochs in the eval_input_reader from maybe 1 to 3. If you are still having trouble, lower the learning_rate.

**Comment by ekesdf** [comment]
thx
